src/applications/coronavirus-chatbot/createCoronavirusChatbot.js

```diff
@@ -24,6 +25,7 @@ export default (_store, widgetType) => {
       });
       window.WebChat.renderWebChat(webchatOptions, root);
     } catch (err) {
+      Sentry.captureException(err);
```

Comment:
It's weird to me that below this there are two different error events logged to GA...should one of them be in a different place in the code?

Reply:
I have the same thoughts, but this was lower on the priority bucket for us. I would think we need to understand why we put two in here before changing them.

Reply:
@shaunhubbard-va is there an ticket created to address that?

Reply:
this was the relevant thread:
https://dsva.slack.com/archives/C011BG82ZL6/p1587498673442600

But I'm not sure why two events are triggered nonetheless.

Reply:
I would like to know how events are created systematically if possible.